Eyansky/Web3-Full-Stack-Hiring
# feature: Check balance on Ethereum Address

#### Description of Task to be completed?
- An input bar where a user can enter in an Ethereum address
- A display that shows the DAI balance for the entered Ethereum address

#### How should this be manually tested?
- Git clone 
`$ git clone https://github.com/Eyansky/Web3-Full-Stack-Hiring/ `
- Checkout develop branch 
`$ git checkout develop`
- Install node modules
`$ npm install`
- Run the app
`$ npm start`
- Input a valid Ethereum address and check if it will present you with the balance. Change the currency as well and test if it gets converted as well.

#### Screenshots (if appropriate)

![Screen Shot 2021-06-26 at 1 34 43 AM](https://user-images.githubusercontent.com/26874561/123493594-99f47180-d625-11eb-9c80-20c2ed82ba3d.png)

- conversion to any currency screenshot image
<img width="1680" alt="Screen Shot 2021-06-26 at 1 35 31 AM" src="https://user-images.githubusercontent.com/26874561/123493605-aa0c5100-d625-11eb-8b46-127605d1a915.png">
